project/CopyTruffleJAR.scala

```diff
@@ -83,6 +83,11 @@ object CopyTruffleJAR {
     libraryUpdates: UpdateReport,
     log: ManagedLogger
   ): Boolean = {
+    val version = System.getProperty("java.version").split("\\.");
+    if (Integer.parseInt(version(0)) >= 21) {
+      // skip all checks on GraalVM for JDK21 and newer
+      return false;
+    }
```

Comment:
What is the reason for this change?

Is SBT now detecting the Truffle modules automatically since 21?

Reply:
There is no `gu` in GraalVM for JDK21+: https://github.com/oracle/graal/issues/6855

Reply:
Ok, I see.

So once we all upgrade to Graal 21, I assume we should be able to remove this line from `.jvmopts`:
```diff
 -Xss16M
 -Xmx4G
 -XX:+UseCompressedOops
---upgrade-module-path=engine/runtime/build-cache/truffle-api.jar
```